packages/components/src/radio-control/index.tsx

```diff
@@ -142,7 +141,16 @@ export function RadioControl(
 					</div>
 				) ) }
 			</VStack>
-		</BaseControl>
+			{ !! help && (
+				<StyledHelp
+					__nextHasNoMarginBottom
+					id={ generateHelpId( id ) }
+					className="components-base-control__help"
+				>
+					{ help }
+				</StyledHelp>
```

Comment:
All this may be a good opportunity to export and reuse directly from `BaseControl`.

Reply:
`StyledHelp` is already being imported from `BaseControl` (it's just not exported publicly, hence why we're not using `BaseControl.StyledHelp`).

I'm hesitant to make changes to `BaseControl`, because I believe we would soon like to take a good look at the component and potentially work on a new version (see https://github.com/WordPress/gutenberg/issues/60836)

Reply:
Sure, but this entire `help` part is just duplicated from `BaseControl` and we could easily reuse. No strong feelings if you are hesitant to touch it though.

Reply:
The part that we're not really extracting and re-using from `BaseControl` is the logic to generate an `id` for the help text, and use it for the `aria-describedby` attribute on the `fieldset` — but it's quite a standard practice, and since we're already going quite custom with `fieldset` and `legend`, I don't think that putting time into this could bring much benefit at this point in time.